workspaces/web/src/components/ChallengePanel.svelte

```diff
@@ -73,12 +76,19 @@
     color: $text-invert;
     background: $yellow;
   }
-
-  .panel .right {
+  .left {
```

Comment:
I wonder if introducing this is really necessary after all. TBH I shouldn't have introduced float here in the first place, it's better to use flexbox layouts IMO.

Reply:
I tried to fix the bug arising from using the paragraph tag present in message detail.
Definitely, a better way to this would be using flexbox or grids.
But as this PR was for a different main issue avoided changing the layout.
let me know if I should replace with flexbox, instead of floating elements

Reply:
I'll accept it like this, but if you have time, can you please create a new issue for refactoring it to flexbox or grid?

Reply:
Sure, no issues.